src/Visualization/Python/PlotSlice.py

```diff
@@ -0,0 +1,250 @@
+        "window server."
+    ),
+)
+@click.option(
+    "--slice-origin",
+    "-C",
+    callback=parse_point,
+    help=(
+        "Coordinates of the center of the slice through the volume "
+        "data. Specify as comma-separated list, e.g. '0,0,0'."
+    ),
+)
+@click.option(
+    "--slice-radius",
+    "-R",
+    type=float,
+    help=(
+        "Visible radius around the center of the slice through the volume data."
+    ),
+)
+@click.option(
+    "--slice-normal",
+    "-n",
+    callback=parse_point,
+    help=(
+        "Direction of the normal of the slice through the volume "
+        "data. Specify as comma-separated list, e.g. '0,0,1' for a slice "
+        "in the xy-plane."
+    ),
+)
+@click.option(
+    "--slice-up",
+    "-u",
+    callback=parse_point,
+    help=(
+        "Up-direction of the slice through the volume "
+        "data. Specify as comma-separated list, e.g. '0,1,0' so the y-axis "
+        "is the vertical axis of the plot."
+    ),
+)
+@click.option(
+    "--num-samples",
+    "-N",
+    type=int,
+    default=200,
+    show_default=True,
+    help=(
+        "Number of uniformly spaced samples along each direction of the slice "
+        "to which volume data is interpolated."
+    ),
+)
+@click.option(
+    "--num-threads",
+    "-j",
+    type=int,
+    show_default="all available cores",
+    help=(
+        "Number of threads to use for interpolation. Only available if compiled"
+        " with OpenMP. Parallelization is over volume data files, so this only"
+        " has an effect if multiple files are specified."
+    ),
+)
+@click.option(
+    "--stylesheet",
+    "-s",
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
+    envvar="SPECTRE_MPL_STYLESHEET",
+    help=(
+        "Select a matplotlib stylesheet for customization of the plot, such "
+        "as linestyle cycles, linewidth, fontsize, legend, etc. "
+        "The stylesheet can also be set with the 'SPECTRE_MPL_STYLESHEET' "
+        "environment variable."
+    ),
+)
+def plot_slice_command(
+    h5_files,
+    subfile_name,
+    obs_id,
+    obs_time,
+    vars,
+    output,
+    slice_origin,
+    slice_radius,
+    slice_normal,
+    slice_up,
+    num_samples,
+    num_threads,
+    stylesheet,
+):
+    """Plot variables on a slice through volume data
+
+    Interpolates the volume data in the H5_FILES to a slice and plots the
+    selected variables. You choose the slice by specifying three corners of a
+    rectangle.
+    """
+    if obs_id is None:
+        raise click.UsageError("Specify '--step' / '-s' or '--time' / '-t'.")
+    if len(vars) > 1:
+        raise click.UsageError("Only one variable can be plotted at a time.")
+    var_name = vars[0]
+
+    # Interpolate the selected variable to the slice
+    if (
+        slice_origin is None
+        or slice_radius is None
+        or slice_normal is None
+        or slice_up is None
+    ):
```

Comment:
Make them click-required options?

Reply:
Then you'd have to specify them even if you don't know the subfile name or the variable name yet and want the available names printed out.